src/cascadia/TerminalSettingsModel/ActionArgs.h

```diff
@@ -308,9 +314,12 @@ namespace winrt::TerminalApp::implementation
     struct SplitPaneArgs : public SplitPaneArgsT<SplitPaneArgs>
     {
         SplitPaneArgs() = default;
-        GETSET_PROPERTY(winrt::TerminalApp::SplitState, SplitStyle, winrt::TerminalApp::SplitState::Automatic);
-        GETSET_PROPERTY(winrt::TerminalApp::NewTerminalArgs, TerminalArgs, nullptr);
-        GETSET_PROPERTY(winrt::TerminalApp::SplitType, SplitMode, winrt::TerminalApp::SplitType::Manual);
+        SplitPaneArgs(SplitState style, Model::NewTerminalArgs terminalArgs) :

```

Comment:
be sure to take winrt class arguments by const reference to avoid additoinal reference counting overhead

Reply:
i wonder if the compiler can see through this and turn it into a `move`...